src/Wallabag/CoreBundle/Resources/views/themes/material/Entry/entry.html.twig

```diff
@@ -156,13 +164,36 @@ main {
             <div class="input-field nav-panel-add-tag" style="display: none">
                 {{ render(controller( "WallabagCoreBundle:Tag:addTagForm", { 'id': entry.id } )) }}
             </div>
-
         </aside>
         <article>
             {{ entry.content | raw }}
         </article>
-    </div>
+        <div id="comments" class="comments-section">
+            <span class="mdi-editor-insert-comment" id="new-comment-button"></span>
+            <div class="nav-panel-add-comment input-field" style="display: none">
+                {{ render(controller( "WallabagCoreBundle:Comment:addCommentForm", { 'id': entry.id } )) }}
+            </div>
+            {% if entry.comments is not empty %}
+                <div id="comments_list" class="mdi-editor-insert-comment">
+                Comments
+                {% for comment in entry.comments %}
+                    <div id="{{ comment.id }}" class="comment">
+                        <hr>
+                        <div class="comment_content">{{ comment.content }}</div>
+                        <div class="comment_infos">
+                            <span class="comment_actions">
+                            <a class="mdi-action-delete" href="{{ path('delete_comment', { 'id': comment.id }) }}" ></a></span>
+                            <a class="return-to-paragraph" href=""><span class="mdi-navigation-arrow-drop-up"> </span></a>
+                            <span class='commentdom'>{{ comment.dom }}</span> 
+                            <span class="comment_metadata"> 
+                            {{ comment.user.username }} - {{  comment.date | date("H:i:s") }} -  {{  comment.date | date("d/m/Y") }}</span>
+                        </div>
+                    </div>
+                {% endfor %}
+                </div>
+            {% endif %}
+        </div>
 {% endblock %}
 
 {% block footer %}
-{% endblock %}
+{% endblock %}
```

Comment:
Do not forget to install a plugin on your IDE to handle `.editorconfig` file to avoid empty line at the end of file.